extensions/wikia/PhalanxII/fallback/Phalanx.class.php

```diff
@@ -414,33 +414,33 @@ static function findBlocked($text, $blocksData, $writeStats = true, &$matchingBl
 				$isCase = $blockData['case'];
 			} else { // short format
 				list( $blockId, $blockText, $blockFlags ) = $blockData;
-				$isRegex = ($blockFlags & self::FLAG_REGEX) > 0;
-				$isExact = ($blockFlags & self::FLAG_EXACT) > 0;
-				$isCase = ($blockFlags & self::FLAG_CASE) > 0;
+				$isRegex = ( $blockFlags & self::FLAG_REGEX ) > 0;
+				$isExact = ( $blockFlags & self::FLAG_EXACT ) > 0;
+				$isCase = ( $blockFlags & self::FLAG_CASE ) > 0;
 			}
 			$origText = $blockText;
 
-			if ($isRegex) {
-				//escape slashes uses as regex delimiter
-				$blockText = str_replace('/', '\/', preg_replace('|\\\*/|', '/', $blockText));
-				if ($isExact) {
-					//add begining and end anchor only once (user might added it already)
-					if (strpos($blockText, '^') !== 0) {
+			if ( $isRegex ) {
+				// escape slashes uses as regex delimiter
+				$blockText = str_replace( '/', '\/', preg_replace( '|\\\*/|', '/', $blockText ) );
+				if ( $isExact ) {
+					// add begining and end anchor only once (user might added it already)
+					if ( strpos( $blockText, '^' ) !== 0 ) {
 						$blockText = '^' . $blockText;
 					}
-					if (substr($blockText, -1) != '$') {
+					if ( substr( $blockText, -1 ) != '$' ) {
 						$blockText .= '$';
 					}
 				}
 				$blockText = "/$blockText/";
-				if (!$isCase) {
+				if ( !$isCase ) {
 					$blockText .= 'i';
 				}
-				//QuickFix™ for bad regexes
-				//TODO: validate regexes on save/edit
+				// QuickFix™ for bad regexes
+				// TODO: validate regexes on save/edit
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)